components/pages/user/ApiTokenInfo.tsx

```diff
@@ -284,6 +284,16 @@ const ApiTokenInfo = () => {
 	const parsedExpiry: number = existingApiToken ? parseExpiry(existingApiToken?.expiryDate) : 0;
 	const tokenIsExpired: boolean = has(existingApiToken, 'expiryDate') && parsedExpiry <= 0;
 
+	const displayToken = (token: ApiToken | null) => {
+		if (!token) {
+			return 'You have no API token...';
+		} else if (existingApiToken?.name) {
```

Comment:
shouldn't this check be on `token` instead?

Reply:
In that case, we either remove EGO support (not doable right now) or acknowledge that the token *may* not include .name (via *optional* typing). That would then enable this...
```suggestion
		} else if (token?.name) {
```

Reply:
updated `ApiToken` interface with optional `name?: string;`